esqueleto.cabal

```diff
@@ -1,6 +1,7 @@
 cabal-version: 1.12
 
 name:           esqueleto
+version:        3.2.0
 version:        3.1.2
```

Comment:
Are there supposed to be two version lines here? What would that mean?

Reply:
Bad merge conflict, thanks for catching!